packages/flutter/lib/src/widgets/sliver_persistent_header.dart

```diff
@@ -224,7 +231,13 @@ class _SliverFloatingPersistentHeader extends _SliverPersistentHeaderRenderObjec
 
   @override
   _RenderSliverPersistentHeaderForWidgetsMixin createRenderObject(BuildContext context) {
-    return new _RenderSliverFloatingPersistentHeaderForWidgets();
+    return new _RenderSliverFloatingPersistentHeaderForWidgets()
```

Comment:
Add a TODO saying that once that bug is fixed we should improve this, maybe? When that bug is fixed I'm going to search the codebase for that issue's URL.